src/main/java/org/mvel2/asm/Opcodes.java

```diff
@@ -58,6 +58,7 @@
     int V1_6 = 0 << 16 | 50;
     int V1_7 = 0 << 16 | 51;
     int V1_8 = 0 << 16 | 52;
+    int V1_9 = 0 << 16 | 53;
```

Comment:
@psiroky The classes of the asm package are just the copy/paste of the original ASM source code (I have no idea of the reason why they decided to go this way), so I am not sure if it is enough to just add a new Opcode here. Probably this change is safe, but in general I don't think we should directly change these classes. It would be a better option if there is a newer version of ASM natively supporting Java 9, so we could copy/paste again its source code here.